openssl/src/x509/mod.rs

```diff
@@ -424,6 +424,22 @@ impl<'a> X509Ref<'a> {
         }
     }
 
+    /// Returns Issuer validity notAfter
+    pub fn not_after(&self) -> Asn1TimeRef {
```

Comment:
It sounds like these are always present, so let's just have an assert that checks for null.